docs/laborok/02-android-ui-compose/index.md

```diff
@@ -292,8 +292,9 @@ fun LoginScreen() {
             label = { Text(passwordLabel) },
             modifier = Modifier.fillMaxWidth(),
             keyboardOptions = KeyboardOptions(
-                keyboardType = KeyboardType.Password
+                keyboardType = KeyboardType.NumberPassword
```

Comment:
Ez is menő, de felesleges ide. Nem pin-t adunk meg, ez egyáltalán nincs jelezve, Password van jelezve.

Reply:
Az eredeti laborban is pin-t akar beiratni, ha nem így szeretnéd akkor persze revertelem

Reply:
Gondoltam az a logikus egy ilyen layoutban, meg szövegkörnyezetben. Köszi, ha reverteled.